src/components/ebay-toast/examples/01-default/template.marko

```diff
@@ -0,0 +1,22 @@
+<ebay-toast a11y-close-text="Close Toast" aria-labelledby="toast-title" open=state.open on-toast-close('closeToast')>
+    <@header><h2 id="toast-title">Heading</h2></@header>
```

Comment:
My concern with this syntax is that the developer has to plumb together the accessibility (i.e. the `aria-labelledby` and the `id`). I believe in some other cases we handle those intricacies behind the scenes...?

Reply:
This is no different than the previous versions. In previous versions we had the title as the first element. 
This on `5.2.x` the version before we split up the header
https://github.com/eBay/ebayui-core/blob/v5.2.3/src/components/ebay-dialog/examples/01-basic/template.marko#L1-L2
Also this is in line with the dialog code. 
Again we should address this in refactors (have a title attribute ideally)

Reply:
Gotcha. Thanks. Will also note this down for a revisit in v6.